docs/Setup.rst

```diff
@@ -136,7 +136,7 @@ Include either ``neo4j`` or ``neo4j-core`` in your ``Gemfile`` (``neo4j`` includ
 
   gem 'neo4j', '~> 7.0.0'
   # OR
-  gem 'neo4j-core', '~> 7.0.0'
+  gem 'neo4j-core', '~> 6.0.0'
```

Comment:
The latest version is 6.1.4.

cf. [neo4j-core](https://rubygems.org/gems/neo4j-core)